framework_tvb/tvb/adapters/datatypes/h5/connectivity_h5.py

```diff
@@ -56,7 +56,7 @@ def get_centres(self):
     def get_region_labels(self):
         return self.region_labels.load()
 
-    def store(self, datatype, scalars_only=False, store_references=False):
+    def store(self, datatype, scalars_only=False, store_references=True):
```

Comment:
@popaula937 I found the signature from parent wrongly copied in few H5File subclasses.
It seems to prevent in some cases (e.g from store_complete) storage of Reference attributes (e.g. ConnectivityMeasure.connectivity).
In some other classes, like here on the ConnectivityH5 the missmatch in signature has no effect, as this entity contains no reference.
I changes in all places of DataTypes the signature, for uniformity, except for Simulator classes (Sim, Burst, Monitors...). Could you pls consider these classes, and judge if   those need changes or not?

Reply:
Sure, looking into it. Thanks for this nice review!

Reply:
Overriding is not necessary for Monitors.
For SimulatorH5 and IntegratorH5, I will do a review soon. Since we now generate the GID on HasTraits, we don't need to wait for their dependencies to be written to H5 before being able to get the GID, so store_reference=False will no longer be necessary.

Reply:
Indeed, the `store_reference=False` flag was making sense when we were not having GID on `HasTraits`, and it had to be filled afterwards. Thanks for explaining why this code!